packages/web/src/common/store/notifications/sagas.ts

```diff
@@ -392,16 +396,42 @@ export function* fetchNotificationUsers(action: FetchNotificationUsers) {
 
 export function* subscribeUserSettings(action: SubscribeUser) {
   const audiusBackendInstance = yield* getContext('audiusBackendInstance')
+  const getFeatureEnabled = yield* getContext('getFeatureEnabled')
+
   yield* call(audiusBackendInstance.updateUserSubscription, action.userId, true)
```

Comment:
yes we still want to make the old call so that we write to both identity and discovery! This is so I can have some time delay between pulling data from identity and actually running the backfill in discovery without worrying about additional writes to identity. after I backfill discovery and enable reads from discovery, I'll remove the old call to identity

Reply:
Ok, perfect. Thanks!